devtools/conda-envs/test-env.yaml

```diff
@@ -35,7 +36,7 @@ dependencies:
     # Optional
   - forcebalance
   - openff-fragmenter-base
-
+  - xtb-python
```

Comment:
This is the only change I'm iffy on (non-blocking). I think it's pretty light but it's only used in ~half of the tests, being dead weight in the others.

Reply:
Yeah, without it we'd need to either add a new environment for integration tests or write some sort of test QC generation harness, neither of which I want to do.

MacOS/RDKit failed a couple of unit tests. I'm re-running it but I'll probably merge even with the unit test failures, coz I'm not sure how this PR could've caused them. Completely agree with you about having the integration test fail.

Reply:
I also see reliably sporadic unit test failures in this codebase (and don't know how to fix them)